### PR TITLE
chore: support localization for extension

### PR DESCRIPTION
**Pull request by HuihuiWu-Microsoft**

### CODE REVIEW

File: packages/vscode-extension/package.nls.json [hunk 1]
```diff
@@ -37,5 +37,227 @@
...
+    "teamstoolkit.commands.feedbackLink.title": "Learn more about help and feedback",
+    "teamstoolkit.common.activate": "Teams Toolkit extension is now active!",
+    "teamstoolkit.common.userCancel": "User Cancel",
+    "teamstoolkit.common.confirm": "Confirm",
+    "teamstoolkit.common.signout": "Sign out",
+    "teamstoolkit.common.signOutOf": "Sign out of '%s'?",
+    "teamstoolkit.common.cancel": "Cancel",
+    "teamstoolkit.common.signin": "Sign in",
+    "teamstoolkit.common.readMore": "Read more",
+    "teamstoolkit.userInterface.noQuestionDescription": "No question description",
```
Comment (swatDong) [inline packages/vscode-extension/package.nls.json:49]: How about considering alpha-bate order for easier maintenance?

**Review by swatDong — DISMISSED** [review]

File: packages/vscode-extension/package.nls.json (hunk 1 above)
Comment (HuihuiWu-Microsoft) [reply packages/vscode-extension/package.nls.json:49]: Do you mean the way as Strings.json? It's required format for VSC extension localization. The old way doesn't work.

Comment (swatDong) [reply packages/vscode-extension/package.nls.json:49]: (My bad typo...)
I mean the alphabetic order for all keys, e.g.,
```
"teamstoolkit.a....",
...
"teamstoolkit.b...",
...
"teamstoolkit.c...",
...
```

Comment (HuihuiWu-Microsoft) [reply packages/vscode-extension/package.nls.json:49]: Good suggestion!

Comment (HuihuiWu-Microsoft) [reply packages/vscode-extension/package.nls.json]: Updated

**Review by swatDong — APPROVED** [review]

**Review by tecton — APPROVED** [review]

**Pull request merged**